tests/test_validators.py

```diff
@@ -1020,11 +1085,11 @@ class Model(BaseModel):
         b: str
 
         @validator('a', allow_reuse=allow_1)
-        def duplicate_name(cls, v):
+        def duplicate_name(cls, v: Any):
```

Comment:
I've noticed you've replaced `v` with `v: Any` in most of these signatures — is that necessary now, or just something you are in the habit of doing? I don't object or anything, just curious

Reply:
Not necessary put since `@validator` has better typing now Pyright will complain at you if the signature is unknown. So I'm mainly adding it for our own sanity.